roles/openshift_logging/tasks/label_node.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- shell: >
+    {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig get node {{host}}
+    --template='{{ '{{index .metadata.labels "' }}{{label}}{{ '"}}' }}'
```

Comment:
s/shell/command/

I would suggest thinking about using -o jsonpath here instead of template to avoid having to escape/quote braces